tools/msvs/msi/product.wxs

```diff
@@ -110,24 +103,18 @@
 
     <Directory Id="TARGETDIR" Name="SourceDir">
       <Directory Id="ProgramMenuFolder">
-        <Directory Id="ApplicationProgramsFolder" Name="io.js"/>
+        <Directory Id="ApplicationProgramsFolder" Name="Node.js"/>
       </Directory>
 
       <Directory Id="$(var.ProgramFilesFolderId)">
-        <Directory Id="INSTALLDIR" Name="iojs">
+        <Directory Id="INSTALLDIR" Name="nodejs">
```

Comment:
Less chance of collision with `nodejs`. Ubuntu distributes as `nodejs` b/c of name collision, even though all its tools are under `node-*` instead of `nodejs-*`.